driver/service.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	storageStateTimeout time.Duration = time.Hour
+	storageStateTimeout time.Duration = time.Millisecond
```

Comment:
Is this correct ? :thinking:  We use this timeout in the SDK to wait storage to enter specific state.

Reply:
Good catch, I had swapped that to something really tiny to test the timeout 🙈 Fixed!